upmc-enterprises/cda2fhir
# UPMCFHIR-246 UPMCFHIR-269 added annotation support for free text instructions in medication supply order

#### What does this PR do?

Adds free text instructions in the dosage field of MedicationStatement

#### Related JIRA tickets:

[UPMCFHIR-246](https://jira.amida.com/browse/UPMCFHIR-246) & [UPMCFHIR-269](https://jira.amida.com/browse/UPMCFHIR-269)

UPMC246 regards Timing in MedicationStatement.dosage, ~however all timing is included in free-text instructions inside the html and can't be pulled out~. Bill also asked for text entries in dosage as well so this should meet that criteria and at least provide some way to access timing data, if only free text. this also wraps up all updates to MedicationStatement.dosage so that finishes up UPMCFHIR269 as well. 

Update: there actually is a timing field, however it does not conform to the CDA spec. I added mapping for it and tests.
#### How should this be manually tested?

Run the integration tests with varios documents and check that the output medication statement dosage has all referenced free text instructions. 

#### Background/Context:

#### New JIRA tickets for clinical review required?
